### DISCUSSION TITLE
data assign to groups

### DISCUSSION THREAD

**Discussion by omid44** [YetiForceCompany/YetiForceCRM#17405] [How to]
i create a page that send message to users and this page is separate from sms sender of yeti , also i create multiple groups and specify data of each groups to just that group (group A just access of data that user of this group insert) but for my sms sender page i want every each group just access their own data and i cant find how data labeled in database , when i see database as example lead section (leadaddress)  all the groups data are in one table (leadaddress)  and don't have any id to specify it , but at the end when all things work correct and all data of each grup just show to that group.
also i found some parts like (groupid ) but i dont see any thing that specify as example leads to that id 
short version: how data labeled in database by groups
